arduino-ide-extension/src/browser/auth/authentication-client-service.ts

```diff
@@ -43,15 +43,15 @@ export class AuthenticationClientService
 
   readonly onSessionDidChange = this.onSessionDidChangeEmitter.event;
 
-  onStart(): void {
+  async onStart(): Promise<void> {
     this.toDispose.push(this.onSessionDidChangeEmitter);
     this.service.setClient(this);
     this.service
       .session()
       .then((session) => this.notifySessionDidChange(session));
 
-    this.setOptions();
-    this.service.initAuthSession()
+    await this.setOptions();
```

Comment:
Theia waits for `onStart` to resolve if it's a promise. We do not need to block the app startup.

Please do:
```ts
this.setOptions().then(() => this.service.initAuthSession());
```

Thoughts?